src/artikler/forside/ForsideBokmal.tsx

```diff
@@ -20,15 +20,15 @@ export const ForsideBokmal: React.FC = () => {
                 <SprakvelgerForside />
 
                 <AlertStripeInfo>
-                    <Lenke href="./korona">
+                    <InternLenke href="/korona">
                         Koronavirus - Flere kan ha rett til økonomisk
                         sosialhjelp
-                    </Lenke>
+                    </InternLenke>
                 </AlertStripeInfo>
 
                 <br />
 
-                <SokOmSosialhjelpPanel href="./slik-soker-du">
+                <SokOmSosialhjelpPanel href="/slik-soker-du?lang=nb">
```

Comment:
Hvorfor trenger vi å legge med språk? Har vi ikke det i coockie/state eller noe?

Reply:
Egentlig bare for å være konsekvent, og litt lettere å copy-paste rundt. På sikt bør vi nok gå over til å ha språk i URL-en, men det er en annen sak.